configure.ac

```diff
@@ -1669,6 +1669,8 @@ else
 fi
 YFLAGS="-d"
 
+TOOL_DEPS_LIBS="${TOOL_DEPS_LIBS} \$(top_builddir)/lib/secret-storage/libsecret-storage.la"
```

Comment:
Do we need to escape $?

Reply:
In `\$(top_builddir)`? Yes, because we want that to appear literally in the Makefile, because we want to defer evaluating that to `make`-time, rather than `configure`-time.

We do not need - nor want to - escape the `$` in `${TOOL_DEPS_LIBS}`, because we want to append to the variable, and for that, we need to evaluate it at configure-time.

Reply:
Thanks for explaining this delayed expansion technique. Sounds good.